openkfw/TruBudget
# 2058 status page redesign

### Checklist




### Description

* redesigned service status page

### How to test

1. run app and go to Service Status page, check that it looks like in new designs in user story




Closes #2058
